tests/examplefiles/nim/example.nim.output

```diff
@@ -911,11 +927,23 @@
 'case'        Keyword
 ' '           Text
 'resp'        Name
-'\n      '    Text
-'of'          Keyword
-' '           Text
-'RESPONSE_ACCEPT' Name
-':'           Punctuation
+'\n      of ' Keyword
+'R'           Name.Label
+'E'           Name.Label
+'S'           Name.Label
+'P'           Name.Label
+'O'           Name.Label
+'N'           Name.Label
+'S'           Name.Label
+'E'           Name.Label
+'_'           Name.Label
+'A'           Name.Label
+'C'           Name.Label
+'C'           Name.Label
+'E'           Name.Label
+'P'           Name.Label
+'T'           Name.Label
```

Comment:
Same as above, this should be a single token.

Reply:
Done.